client/post-editor/editor-publish-date/post-scheduler.jsx

```diff
@@ -89,12 +82,11 @@ export default class PostScheduler extends PureComponent {
 		};
 
 		return (
-			<span className="editor-ground-control__schedule-post">
+			<span className="editor-publish-date__post-scheduler">
```

Comment:
This class name is unused, I'll probably remove it and the wrapper element when I address the current round of feedback.

Reply:
I thought I would be able to remove the wrapper element in favor of returning an array, but:

```
Uncaught Error: PostScheduler.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.
```

So I've changed this to a plain `<div>` instead, which is more common for an element that is only needed as a wrapper.